pages/login.php

```diff
@@ -98,48 +102,51 @@
       <?php include("../templates/navBar.php") ?>
   </div>
 
-	<form action="../include/login.inc.php" method="POST">
```

Comment:
why is this code deleted and added again?

Reply:
Might be due to the `phpfmt`

Reply:
Actually i have put the form inside a div with class container.